### PR TITLE
OCP 4.15 / OLM 1.0 Release notes

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.15

Issue: [OSDOCS-9730](https://issues.redhat.com/browse/OSDOCS-9730)

Link to docs preview: [OLM 1.0 (tech preview) release notes](https://71900--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-olmv1)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -513,6 +513,23 @@ If the following conditions are true, the feature is enabled by default:
...
+
+Support for version ranges::
+You can specify a version range by using a comparison string in an Operator or extension’s custom resource (CR). If you specify a version range in the CR, OLM 1.0 installs or updates to the latest version of the Operator that can be resolved within the version range. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.html#olmv1-updating-an-operator_olmv1-installing-operator[Updating an Operator] and xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.html#olmv1-version-range-support_olmv1-installing-operator[Support for version ranges]
+
+Performance improvements in the Catalog API::
+The Catalog API now uses a HTTP service to serve catalog content on the cluster. Previously, custom resource definitions (CRDs) were used for this purpose. The transition to using a HTTP service to serve catalog content reduces the load on the Kube API server. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-finding-operators-to-install_olmv1-installing-operator[Finding Operators to install from a catalog]
+
+include::snippets/olmv1-cli-only.adoc[]
```
Comment (michaelryanpeter) [inline release_notes/ocp-4-15-release-notes.adoc:529]: Note to reviewers:
The fixup for the OCP version is in PR https://github.com/openshift/openshift-docs/pull/71901

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -513,6 +513,23 @@ If the following conditions are true, the feature is enabled by default:
 [id="ocp-4-15-olm"]
 === Operator lifecycle
 
+[id="ocp-4-15-olmv1"]
+==== {olmv1-first} (Technical Preview)
+
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release. {product-title} 4.14 introduced components for a next-generation iteration of OLM as a Technology Preview feature, known during this phase as _{olmv1}_. This updated framework evolves many of the concepts that have been part of previous versions of OLM and adds new capabilities.
+
+During this Technology Preview phase of {olmv1} in {product-title} {product-version}, administrators can explore the following features:
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:521]:
```suggestion
During this Technology Preview phase of {olmv1} in {product-title} {product-version}, administrators can explore the following features added with this release:
```
or similar?

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -513,6 +513,23 @@ If the following conditions are true, the feature is enabled by default:
...
+
+Support for version ranges::
+You can specify a version range by using a comparison string in an Operator or extension’s custom resource (CR). If you specify a version range in the CR, OLM 1.0 installs or updates to the latest version of the Operator that can be resolved within the version range. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.html#olmv1-updating-an-operator_olmv1-installing-operator[Updating an Operator] and xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.html#olmv1-version-range-support_olmv1-installing-operator[Support for version ranges]
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:524]:
```suggestion
You can specify a version range by using a comparison string in an Operator or extension's custom resource (CR). If you specify a version range in the CR, {olmv1} installs or updates to the latest version of the Operator that can be resolved within the version range. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-updating-an-operator_olmv1-installing-operator[Updating an Operator] and xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-version-range-support_olmv1-installing-operator[Support for version ranges].
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -513,6 +513,23 @@ If the following conditions are true, the feature is enabled by default:
...
+
+Performance improvements in the Catalog API::
+The Catalog API now uses a HTTP service to serve catalog content on the cluster. Previously, custom resource definitions (CRDs) were used for this purpose. The transition to using a HTTP service to serve catalog content reduces the load on the Kube API server. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-finding-operators-to-install_olmv1-installing-operator[Finding Operators to install from a catalog]
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:527]:
```suggestion
The Catalog API now uses an HTTP service to serve catalog content on the cluster. Previously, custom resource definitions (CRDs) were used for this purpose. The transition to using an HTTP service to serve catalog content reduces the load on the Kubernetes API server. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-finding-operators-to-install_olmv1-installing-operator[Finding Operators to install from a catalog].
```

**Review by m1kola — APPROVED** [review]
/lgtm

**Pull request merged**